esileapclient/tests/unit/common/test_utils.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+from esileapclient.common import utils
+
+
+class TestUtils(unittest.TestCase):
+
+    def setUp(self):
+        self.nodes = [
+            {'properties': {'cpus': '40', 'memory_mb': '131072'}},
+            {'properties': {'cpus': '80', 'memory_mb': '262144'}},
+            {'properties': {'cpus': '20', 'memory_mb': '65536'}},
+        ]
+        self.parser = utils.PropertyFilterParser()
+
+    def test_convert_value(self):
+        self.assertEqual(self.parser.convert_value('10'), 10)
+        self.assertEqual(self.parser.convert_value('10.5'), 10.5)
```

Comment:
You want to test each possible path through a function, so you'll want to have an additional test here that verifies that a string is returned as a string